### PR TITLE
Test texture deletion after CopyTexImage2D/CopyTexSubImage2D.

### PR DESCRIPTION
**Pull request by kenrussell**
Regression test for http://anglebug.com/4267 and linked bugs.

Added to, and re-alphabetized, test list.

### CODE REVIEW

**Comment by kenrussell** [comment]
Per discussion at last F2F.

@kainino0x @grorg @RafaelCintron @shrekshao in particular, FYI, and any feedback welcome.

File: sdk/tests/conformance/textures/misc/texture-copying-and-deletion.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+debug('deleting texture');
+gl.deleteTexture(texture);
+debug('deleting texture2');
+// On buggy driver, crashes during this deleteTexture call.
+gl.deleteTexture(texture2);
+
+setTimeout(function() {
+  shouldBe("contextWasLost", "false");
+  finishTest();
+}, 1000);
```
Comment (jdashg) [inline sdk/tests/conformance/textures/misc/texture-copying-and-deletion.html]: Do we have to wait a whole second?

Comment (kenrussell) [reply sdk/tests/conformance/textures/misc/texture-copying-and-deletion.html]: The delay could probably be reduced, but wanted to maximize the chances of catching GPU process crashes on older, slower machines.

**Comment by kenrussell** [comment]
Thanks for your review. Merging now.

**Pull request merged**